catalog/async/catalog-async-data-api/src/main/java/org/codice/ddf/catalog/async/data/api/internal/ProcessResource.java

```diff
@@ -61,9 +61,12 @@
   String getMimeType();
 
   /**
-   * Return the input stream containing the {@link ProcessResource}'s actual data content.
+   * Return an input stream containing the {@link ProcessResource}'s actual data content. When this
+   * method is called it creates a {@code TemporaryFileBackedOutputStream} so that it can produce a
```

Comment:
✏️ This is an interface so indicating the implementation will use a `TemporaryFileBackedOutputStream` is leaking info which you do not know here.